modules/user0_webauth/default/htaccess.txt

```diff
@@ -0,0 +1,20 @@
+# $Id$
+ErrorDocument 401 /user
+
+AuthType WebAuth
+require valid-user
+WebAuthOptional off
+
+WebAuthLdapAttribute suAffiliation
+WebAuthLdapAttribute mail
+WebAuthLdapAttribute uid
+WebAuthLdapAttribute displayName
+WebAuthLdapAttribute ou
+WebAuthLdapAttribute telephonenumber
+WebAuthLdapAttribute facsimiletelephonenumber
+WebAuthLdapAttribute postaladdress
+
+WebAuthLdapPrivgroup stanford:student
```

Comment:
How do these interact with the restrictions in the webauth grants? Do they bypass them? When do these get invoked?